src/index.ts

```diff
@@ -4,11 +4,12 @@ import * as ios from "./ios";
 
 export interface Config extends Partial<android.Config>, Partial<ios.Config> {
   icon: Partial<input.Config>;
+  exclude?: { android: boolean; ios: boolean };
 }
 
 export async function* generate(config: Config): AsyncIterable<string> {
   const iconInput = await input.readIcon(config.icon);
 
-  yield* android.generate(config, iconInput);
-  yield* ios.generate(config, iconInput);
+  if (!config.exclude?.android) yield* android.generate(config, iconInput);
```

Comment:
While not formally defined, the repository code style uses brackets around if single line statements. Would be nice to include them here as well.